packages/brave.rb

```diff
@@ -3,17 +3,19 @@
 class Brave < Package
   description 'Next generation Brave browser for macOS, Windows, Linux, Android.'
   homepage 'https://brave.com/'
-  version '1.28.33'
```

Comment:
Seems this is a nightly version, can we revert back to the latest stable?

Reply:
The last version was nightly too.

Reply:
I reverted to pre-release 1.27.106.